src/main/java/org/madhuri/app/message/model/Channel.java

```diff
@@ -44,33 +44,43 @@ public class Channel {
 
 	@Column(name = "updated_by")
 	private Long updatedBy;
-
-	@ManyToMany(fetch = FetchType.LAZY)
-	@JoinTable(name = "channel_user", joinColumns = @JoinColumn(name = "channel_id"), inverseJoinColumns = @JoinColumn(name = "user_id"))
-	private List<User> users = new ArrayList<>();
+	
+	@ManyToMany(fetch = FetchType.EAGER)
+    @JoinTable(
+            name = "channel_user",
+            joinColumns = @JoinColumn(name = "channel_id"),
+            inverseJoinColumns = @JoinColumn(name = "user_id")
+        )
+        private List<User> users = new ArrayList<>();
```

Comment:
why do we want to fetch all the users for the channel every time we fetch channel details ?